Apps/Contoso.Forms.Demo/Contoso.Forms.Demo.UWP/Contoso.Forms.Demo.UWP.csproj

```diff
@@ -11,7 +11,7 @@
     <AssemblyName>Contoso.Forms.Demo.UWP</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion>10.0.17763.0</TargetPlatformVersion>
+    <TargetPlatformVersion>10.0.18362.0</TargetPlatformVersion>
```

Comment:
Update TargetPlatformVersion to 10.0.18362.0 due to my OS version. Not sure if it can work by AC service. Otherwise, I can revise it.

Reply:
Yes, please revert it for now. (it is not OS version, the target SDK can installed via VS installer)

Reply:
Yes, please revert it for now. (it is not OS version, the target SDK can installed via VS installer)

Reply:
I see. Thanks. Revert in new commit. Please help to review again.